src/components/experimental/Calendar/Calendar.tsx

```diff
@@ -1,84 +1,103 @@
-import React, { ReactElement } from 'react';
-import {
-    Calendar as BaseCalendar,
-    CalendarProps as BaseCalendarProps,
-    RangeCalendarProps,
-    CalendarGridHeader,
-    CalendarGridBody,
-    DateValue,
-    RangeCalendar
-} from 'react-aria-components';
+import React, { useRef, useEffect } from 'react';
+import { DayPicker, DayButton, getDefaultClassNames } from 'react-day-picker';
+import { format } from 'date-fns';
 import ChevronLeftIcon from '../../../icons/arrows/ChevronLeftIcon';
 import ChevronRightIcon from '../../../icons/arrows/ChevronRightIcon';
 
 import * as Styled from './Calendar.styled';
 
-type CalendarProps = { visibleMonths?: 1 | 2 | 3 } & (
-    | ({ selectionType?: 'single' } & Omit<BaseCalendarProps<DateValue>, 'visibleDuration'>)
-    | ({ selectionType: 'range' } & Omit<RangeCalendarProps<DateValue>, 'visibleDuration'>)
-);
+type Props = React.ComponentProps<typeof DayPicker> & {
+    selectionType?: 'single' | 'range' | 'multiple';
+    visibleMonths?: 1 | 2 | 3;
+};
 
 function Calendar({
-    value,
-    minValue,
-    defaultValue,
-    maxValue,
-    onChange,
+    className,
```

Comment:
why do we have both `className` and `classNames` props? Seem quite similar to me.. if we need both, can we name the props differently depending on their purpose to help us distinguish them better?